{"name": "Проверка ошибки при авторизации с несуществующим курьером", "status": "passed", "steps": [{"name": "Логин курьера в систему", "status": "passed", "parameters": [{"name": "data", "value": "{'login': 'pherring', 'password': <bound method Provider.password of <faker.providers.misc.en_US.Provider object at 0x000001134296B2F0>>}"}], "start": 1714844799683, "stop": 1714844800375}, {"name": "Проверка статуса и текста", "status": "passed", "parameters": [{"name": "code", "value": "404"}, {"name": "name", "value": "'message'"}, {"name": "text", "value": "'Учетная запись не найдена'"}], "start": 1714844800375, "stop": 1714844800375}], "start": 1714844799683, "stop": 1714844800377, "uuid": "32e52a8c-a13b-471a-9f14-cee8c292e4d3", "historyId": "fb492307a2b46644b5afd69ba7ac928d", "testCaseId": "fb492307a2b46644b5afd69ba7ac928d", "fullName": "tests.test_courier_login.TestCourierLogin#test_non_existent_courier_login_failed", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_courier_login"}, {"name": "subSuite", "value": "TestCourierLogin"}, {"name": "host", "value": "Sasaraf"}, {"name": "thread", "value": "19964-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_courier_login"}]}